src/i18n/strings/ru.json

```diff
@@ -0,0 +1,307 @@
+{
+    "accept": "принимать",
+    "accepted an invitation": "принял приглашение",
+    "accepted the invitation for": "принял приглашение на",
+    "Account": "Аккаунт",
+    "Add email address": "Добавить email адрес",
+    "Add phone number": "Добавить телефонный номер",
+    "Admin": "Admin",
+    "Advanced": "Расширенный",
+    "Algorithm": "Алгоритм",
+    "all room members": "все участники комнаты",
+    "all room members, from the point they are invited": "все участники комнаты, с момента приглашения",
+    "all room members, from the point they joined": "все участники комнаты, с момента вступления",
+    "an address": "адрес",
+    "and": "и",
+    "An email has been sent to": "Email был отправлен",
+    "A new password must be entered.": "Введите новый пароль.",
+    "answered the call.": "принятый звонок.",
+    "anyone.": "кто угодно",
+    "Anyone who knows the room's link, apart from guests": "Любой, кто знает ссылку на комнату, кроме гостей",
+    "Anyone who knows the room's link, including guests": "Любой, кто знает ссылку комнаты, включая гостей",
+    "Are you sure you want to leave the room?": "",
+    "Are you sure you want to reject the invitation?": "Вы уверены что вы хотите отклонить приглашение?",
+    "Are you sure you want upload the following files?": "Вы уверены что вы хотите закачать следующий файл?",
+    "banned": "banned",
+    "Banned users": "Запрещенный пользователь",
+    "Bans user with given id": "Запретить пользователя с определенным id",
+    "Blacklisted": "В черный список",
+    "Bug Report": "Отчет ошибок",
+    "Bulk Options": "Объемные параметры",
+    "Can't connect to homeserver - please check your connectivity and ensure your": "Не может соеденится с домашним сервером - пожалуйста убедитесь в соеденение ",
+    "Can't connect to homeserver via HTTP when an HTTPS URL is in your browser bar": "Не может соеденится с домашним сервером по HTTP  когда HTTPS URL в вашей адрессной сроке",
+    "Can't load user settings": "Не может загрузить настройки пользователя",
+    "changed avatar": "изменен аватар",
+    "changed name": "измененное имя",
+    "changed their display name from": "changed their display name from",
+    "changed their profile picture": "changed their profile picture",
+    "changed the power level of": "changed the power level of",
+    "changed the room name to": "changed the room name to",
+    "changed the topic to": "changed the topic to",
+    "Changes to who can read history will only apply to future messages in this room": "",
+    "Changes your display nickname": "",
+    "Claimed Ed25519 fingerprint key": "",
+    "Clear Cache and Reload": "",
+    "Clear Cache": "",
```

Comment:
Can you remove this empty translations? Not that the string is missing completely later...